src/Metadata/Property/PropertyMetadata.php

```diff
@@ -60,7 +60,7 @@ public function getType()
      *
      * @param Type $type
      *
-     * @return self
+     * @return PropertyMetadata
```

Comment:
Those terms are defined in PSR-5 (still a draft): https://github.com/phpDocumentor/fig-standards/blob/b994d853b8ba33bd4bfb00e53cc268d57a5d1e7b/proposed/phpdoc.md#keyword